src/ssl_load.c

```diff
@@ -5178,6 +5196,14 @@ int wolfSSL_CTX_set_default_verify_paths(WOLFSSL_CTX* ctx)
     #endif
     }
 
+#ifdef XGETENV
+    if (certFile != NULL) {
```

Comment:
XFREE checks for NULL - no need to do it here.

Reply:
I don't think that's true or else why would we do [this](https://github.com/wolfSSL/wolfssl/blob/master/linuxkm/lkcapi_glue.c#L2454) among other places in the code. [Here](https://github.com/wolfSSL/wolfssl/blob/master/wolfcrypt/src/memory.c#L414) is an example where we don't check to make sure it's not NULL. It is used when WOLFSSL_XFREE_NO_NULLNESS_CHECK is defined.

Reply:
*sigh* this is one that has been going on for a long time.
At one time we changed all the XFREE macros to check for NULL before calling free or whatever for the platform.
I'm not sure but I think WOLFSSL_XFREE_NO_NULLNESS_CHECK is used to remove the check as the free implementation does it.